crates/levm/src/program.rs

```diff
@@ -0,0 +1,47 @@
+use std::collections::HashSet;
+
+use bytes::Bytes;
+
+use crate::operations::Operation;
+
+pub struct Program {
+    pub operations: Vec<Operation>,
+    pub jumptable: HashSet<usize>,
```

Comment:
I suggest we drop the `Program` approach since it adds some unneeded complexity and also limitations. When checking jumpdests, we can just check the jump target is a jumpdest opcode.

Reply:
Okay, just deleted the `Program` struct and returned to the `Bytes` one 🫡

Reply:
> I suggest we drop the Program approach since it adds some unneeded complexity and also limitations. When checking jumpdests, we can just check the jump target is a jumpdest opcode.

Yep, I agree with this.